Framework/Algorithms/src/NormaliseByCurrent.cpp

```diff
@@ -57,6 +57,8 @@ double NormaliseByCurrent::extractCharge(
                            "wrong current.\n";
   }
   // Handle multiperiod data.
+  // The number of periods is set above by reference
```

Comment:
The current function works, so I don't object to suppressing this warning. I'm also not sure if manually specifying the template argument is better or worse than having an in/out parameter.